yt_dlp/extractor/omnyfm.py

```diff
@@ -0,0 +1,64 @@
+import functools
+
+from .common import InfoExtractor
+from ..utils import (
+    OnDemandPagedList,
+    clean_html,
+    float_or_none,
+    int_or_none,
+    str_or_none,
+    traverse_obj,
+    unified_strdate,
+    url_or_none,
+)
+
+
+class OmnyFMShowIE(InfoExtractor):
+    IE_NAME = 'omnyfm:show'
+    _VALID_URL = r'https?://omny\.fm/shows/(?P<id>[\w-]+)'
+    _EMBED_REGEX = [r'<iframe[^>]+?src=(?:["\'])(?P<url>https?://omny\.fm/shows/.+?)\1']
```

Comment:
The `\1` reference does not work.

```suggestion
    _EMBED_REGEX = [r'<iframe[^>]+?src=(["\'])(?P<url>https?://omny\.fm/shows/.+?)\1']
```

Reply:
Wow! Did not know that was the cause behind EMBED functionality not working. Cant thank enough. I've fixed it